apps/builddao/widget/components/modals/CreateProposal.jsx

```diff
@@ -1,9 +1,11 @@
-const { daos } = VM.require("${config_account}/widget/fetch.daos") || {
-  daos: [],
-};
-
+const DaoSDK = VM.require("sdks.near/widget/SDKs.Sputnik.DaoSDK") || (() => {});
+const sdk = DaoSDK("build.sputnik-dao.near");
```

Comment:
Perhaps this could also be added to aliases?
```suggestion
const sdk = DaoSDK("${alias_buildDAO}");
```